src/App.jsx

```diff
@@ -1,15 +1,29 @@
-import { Link } from "react-router-dom";
+import React from 'react'
+import BackcountryLogo from '../src/logos/BackCountry-logo.svg'
+import { useNavigate } from 'react-router-dom';
 
 function App() {
+  const navigate = useNavigate();
   const env = process.env.NODE_ENV;
   return (
-    <div className="m-[40px]">
-      <p>
-        <Link to="sso">Log in with SSO</Link>
-        <p>Environment: {env}</p>
-      </p>
+    <div>
+      <section className="bg-gray-50">
+      <div className="flex items-center justify-center mx-auto md:h-screen">
+      <div className="w-full bg-white rounded-lg shadow md:mt-0 sm:max-w-md xl:p-0">
```

Comment:
why are u adding css for md, sm, xl
It's not part of the requirement.

Reply:
Remove such code everywhere.

Reply:
addressed